packages/lending/src/protocols/compound-v3/lending-protocol.ts

```diff
@@ -321,8 +322,8 @@ export class LendingProtocol extends Protocol {
     ];
     for (const { token, borrowCollateralFactor, liquidateCollateralFactor, supplyCap, totalSupply } of assets) {
       const apy = '0';
```

Comment:
Do you think it's a good way that:
1. define a constant in config and use it to illustrate that compound v3 does not have any apy for supplying collaterals.
2. add a comment to explain why we give 0 here.

Which way do you prefer?